controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/ApplicationSerializer.java

```diff
@@ -179,7 +179,6 @@ public Slime toSlime(Application application) {
         application.projectId().ifPresent(projectId -> root.setLong(projectIdField, projectId));
         application.deploymentIssueId().ifPresent(jiraIssueId -> root.setString(deploymentIssueField, jiraIssueId.value()));
         application.ownershipIssueId().ifPresent(issueId -> root.setString(ownershipIssueIdField, issueId.value()));
-        root.setBool(builtInternallyField, application.internal());
```

Comment:
This will cause `internal` to become `false` for all apps on the remaining 2/3 controllers once the first upgrades. Is this fine? If not, keep writing this as `true` for at least one version.

Reply:
Ooooooh, good spot. That would have been painful.